File: Python/day_4_only_floats.py
def only_floats(a,b):
    print(a,b)
    if ((type(a)==float)and(type(b))==float):
        return 2
    if (type(a)==float):
        return 1
    if (type(b)==float):
        return 1
    else:
        return 0
output=only_floats(12,23.4)
print(output)

File: JavaScript/day_4_only_floats.js
function only_floats(a, b){
    if (((a%1)!=0)&&((b%1)!=0)){
        return 2;
    }
    if (((a%1)!=0)||((b%1)!=0)){
        return 1;
    }
    //if ((b%1)!=0){
      //  return 1;
    //}
    else{
        return 0;
    }
}
let a=12 ;
let b=23;
let output=only_floats(a,b);
console.log(output);